flow/layers/container_layer_unittests.cc

```diff
@@ -192,6 +196,165 @@ TEST_F(ContainerLayerTest, NeedsSystemComposite) {
                                                child_path2, child_paint2}}}));
 }
 
+TEST_F(ContainerLayerTest, RasterCacheTest) {
```

Comment:
@flar Hi, I add a test case which can describe current raster cache algorithm; Hop you review it thanks